cypress/integration/computation_modern_combinations.spec.ts

```diff
@@ -0,0 +1,78 @@
+/// <reference types="cypress" />
+
+describe("ComputationMenu", () => {
+    beforeEach(() => {
+        cy.clearLocalStorage();
+        cy.viewport(1080, 720);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['cy' is not defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5611900)